src/main/java/Duke.java

```diff
@@ -1,10 +1,119 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static class Task {
+        protected String description;
+        protected boolean isDone;
+
+        public Task(String description) {
+            this.description = description;
+            this.isDone = false;
+        }
+
+        public String getStatusIcon() {
+            return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
+        }
+        public void markAsDone(){
+            this.isDone = true;
+        }
+        public String toString() {
+            return description;
+        }
+    }
+
+    public static class Todo extends Task{
```

Comment:
some methods have a space between method name and the open brackets while others doesnt. ie. "Todo {" versus "Todo{". A solution would be to use Ctrl + Alt + L in Intellij to reformat the code easily.